index.html

```diff
@@ -103,25 +121,44 @@ <h2>PORTFOLIO</h2>
 			</div>
 			
 		</section>
-		
 	</main>
+
 	<footer class="contact" id="contact">
 		<div class= contact-wrapper>
```

Comment:
You forgot quotation marks here! and there's a space before the class name

Reply:
Fixed! thanks